### PR TITLE
John/auth

### PR DESCRIPTION
**Pull request by JohnGordinier**
The data is updated with Login Authorization using bcrypt and tokens. The CRUD Routes have been created in the Server with add, update, delete requiring token access from admin.

### CODE REVIEW

**Comment by gnprwx** [comment]
Merge conflicts have been resolved. Ready for review.

**Review by gnprwx — CHANGES_REQUESTED** [review]
Looks like .env.template was deleted in this PR. Can you put it back in/create a new one with the correct variables to be used and push it? Thanks.

**Comment by gnprwx** [comment]
We have AdminLogin and Modal with the same idea of providing login. Are these solving different problems or the same and Modal component is a dummy login until AdminLogin works as intented (if it doesn't already)?

File: client/src/Landing.jsx
Comment (gnprwx) [inline client/src/Landing.jsx:1]: This is old and needs to be updated to Mark's code. This should not be in Landing. @JohnGordinier get with @mnorthup28 on this.

**Comment by gnprwx** [comment]
Looks like this PR needs work. Otherwise it's going to overwrite @mnorthup28 work and undo the changes I had requested. Closing until @JohnGordinier can get with @mnorthup28 to figure out the Modal situation as the code here is going to actively cause regression.

**Comment by JohnGordinier** [comment]
@mnorthup28 code was pulled off of my branch. What happened in the last 30 minutes?

**Pull request closed without merging**